modules/tinymce/src/themes/silver/main/ts/ui/icons/Icons.ts

```diff
@@ -17,8 +19,26 @@ const getOr = (name: string, icons: IconProvider, fallback: Optional<string>): s
 
 const getFirst = (names: string[], icons: IconProvider): string => Arr.findMap(names, (name) => Optional.from(icons()[name.toLowerCase()])).getOrThunk(() => defaultIcon(icons));
 
+const render = (tagName: string, iconHtml: string, classes: string[], behaviours: Partial<SimpleOrSketchSpec> = {}): SimpleOrSketchSpec => {
+  return ({
+    dom: {
+      tag: tagName,
+      innerHtml: iconHtml,
+      classes
+    },
+    events: AlloyEvents.derive([
+      AlloyEvents.runOnAttached((comp) => {
+        // set focusable=false on SVGs to prevent IE 11 from focusing the toolbar when tabbing into the editor
+        SelectorFind.child(comp.element, 'svg').each((svg) => Attribute.set(svg, 'focusable', 'false'));
```

Comment:
I don't think there's any scenario where we could end up with multiple SVGs in the same icon element, but is it worth doing this just to be defensive?
```suggestion
        const svgs = SelectorFilter.children(comp.element, 'svg');
        Arr.each(svgs, (svg) => Attribute.set(svg, 'focusable', 'false'));
```

Reply:
We only support a single icon, so I don't want to be encouraging or looking like we support this.